### ISSUE TITLE
[Bug] 1.6.3 regression — can't click in note after app is activated via shortcut

### ISSUE THREAD

**Issue by jeff-h** [glushchenko/fsnotes#237]
As per title, when I hide the app and then activate it using the "Search shortcut", I cannot click inside the displayed note to position the cursor.

**Comment by glushchenko** [comment]
Try build 139 https://github.com/glushchenko/fsnotes/releases/download/1.6.3/FSNotes_1.6.3_build139.zip

**Comment by jeff-h** [comment]
Unfortunately that crashes on launch for me. Looks like a code signing issue?

```
Crashed Thread:        0

Exception Type:        EXC_CRASH (Code Signature Invalid)
Exception Codes:       0x0000000000000000, 0x0000000000000000
Exception Note:        EXC_CORPSE_NOTIFY

Termination Reason:    Namespace CODESIGNING, Code 0x1
```

Sorry for the nuisance.

**Comment by glushchenko** [comment]
Please try to redownload.

**Comment by jeff-h** [comment]
Same issue :(

**Comment by gingerbeardman** [comment]
OK for me in 1.6.3 (140)

**Comment by jeff-h** [comment]
1.6.3 (140) also works for me. Thanks for the fix!

**Comment by jeff-h** [comment]
This bug has reappeared for me, since v2.0

**Comment by glushchenko** [comment]
@jeff-h сan you help me with reproduction?

**Comment by jeff-h** [comment]
I can try :) Follow these steps:

* select a note
* hide the app
* activate the app using the "Search shortcut" (set in preferences)
* try to click inside the note body

At this point I'm not able to position the text cursor anywhere in the note body. I have to select a different note before I can place the text cursor in the note.

**Comment by glushchenko** [comment]
Fixed in 2.0.3

**Comment by jeff-h** [comment]
Fixed!
